src/AwsLambda/AwsLambdaOpenTracer/Tracer.csproj

```diff
@@ -8,10 +8,10 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Amazon.Lambda.Core" Version="1.0.0" />
-    <PackageReference Include="Amazon.Lambda.Serialization.Json" Version="1.1.0" />
-    <PackageReference Include="OpenTracing" Version="0.12.0" />
-    <PackageReference Include="ILRepack" Version="2.0.16" />
+    <PackageReference Include="Amazon.Lambda.Core" Version="2.1.0" />
+    <PackageReference Include="Amazon.Lambda.Serialization.Json" Version="2.1.0" />
```

Comment:
It appears that this library is not directly referencing or utilizing Amazon.Lambda.Serialization.Json.  Lambda is now moving towards using Amazon.Lambda.Serialization.SystemTextJson as the default serializer.  If it is correct that you are not using this reference, can it simply be removed?  I have submitted #1189 with that exact change.